package.json

```diff
@@ -53,6 +53,7 @@
     "morgan": "1.7.0",
     "pelias-categories": "1.1.0",
     "pelias-config": "2.3.0",
+    "pelias-labels": "^1.3.0",
```

Comment:
This has to be an exact version (no caret) so that we can track all the changes in git commits.